webrtc-c/canary/jobs/runner.groovy

```diff
@@ -113,6 +115,7 @@ pipeline {
         string(name: 'MASTER_NODE_LABEL')
         string(name: 'VIEWER_NODE_LABEL')
         string(name: 'RUNNER_LABEL')
+        string(name: 'SCENARIO_LABEL')
```

Comment:
Do we need a new label? Why not use `RUNNER_LABEL` itself?

Reply:
`RUNNER_LABEL` is more granular and it's being used for the channel name. e.g. RUNNER_LABEL can be "WebrtcPeriodicLowBitrateEC2" and `SCENARIO_LABEL` can be "WebrtcPeriodic"